tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -199,8 +199,10 @@ def test_positive_smoke(self):
                     common_locators['alert.success']
                 ))
             # Add puppet-module to content-view
-            self.content_views.add_puppet_module(
-                cv_name, 'httpd', filter_term='Latest')
+            if not bz_bug_is_open(1297308):
```

Comment:
Have been a long time that we are not blocking API, CLI and UI tests if smoke is not green. With that said we can conclude that it is a long time that we don't get a green build for smoke.

Unfortunately py.test is not able to consider a subtest which will not allow us to just skip a portion of it.